providers/implementations/kdfs/hkdf.c

```diff
@@ -87,6 +88,7 @@ typedef struct {
     size_t data_len;
     unsigned char *info;
     size_t info_len;
+    int pedantic;
```

Comment:
This might be better conditioned on FIPS_MODULE.

Reply:
Made it be conditioned on FIPS_MODULE.